xaitk_saliency_demo/app/ml/models.py

```diff
@@ -103,6 +107,44 @@ def model(self):
         return self._model
 
 
+class ImagenetModel(AbstractModel):
```

Comment:
ImageNetModel ?

Reply:
that bothered me to, but I decided to follow the `imagenet_categories, imagenet_model_loader`  scake_case that is elsewere in the codebase